core/field_angle.js

```diff
@@ -51,35 +52,6 @@ Blockly.FieldAngle = function(text, opt_validator) {
 goog.inherits(Blockly.FieldAngle, Blockly.FieldTextInput);
 
 /**
- * Sets a new change handler for angle field.
- * @param {Function} handler New change handler, or null.
- */
-Blockly.FieldAngle.prototype.setValidator = function(handler) {
```

Comment:
The call is in the super-class constructor:
https://github.com/google/blockly/blob/master/core/field.js#L50

The purpose of this function was to override the old `setValidator`:
https://github.com/google/blockly/blob/master/core/field.js#L227

and to wrap it with the angleValidator. I don't want to override field's `setValidator`, and getting rid of the override does the job I think.